pump/storage/storage_test.go

```diff
@@ -277,15 +277,15 @@ func (as *AppendSuit) TestDoGCTS(c *check.C) {
 	defer cleanAppend(append)
 
 	var i int64
-	var n int64 = 1 << 11
+	var n int64 = 1024 * 500
```

Comment:
I think we can use `1024 * 500 + 500` rather than `1024 * 500` to test branch of line 727 in `storage.go`.